lib/iris/fileformats/nimrod.py

```diff
@@ -296,7 +328,9 @@ def load_cubes(filenames, callback=None):
                         # End of file. Move on to the next file.
                         break
 
-                    cube = iris.fileformats.nimrod_load_rules.run(field)
+                    cube = iris.fileformats.nimrod_load_rules.run(
+                        field, handle_metadata_errors=handle_metadata_errors
```

Comment:
For the load api extension proposed in #3720, this isn't the correct syntax.
Instead, it will need to set a specific `loader_kwargs` keyword,  
e.g. in this case `..., loader_kwargs={'handle_metadata_errors':handle_metadata_errors})'